File: Palindrome check.cs
using System;

public class Program
{
	static void Main()
	{
		Console.WriteLine("Enter a word:");
		string input = Console.ReadLine();

		char[] charArray = input.ToCharArray();
		Array.Reverse(charArray);
		string reversed = new string(charArray);

		bool isPalindrome = input == reversedInput;

		if (isPalindrome)
		{
			Console.WriteLine("The input is a palindrome.");
		}
		else
		{
			Console.WriteLine("The input is not a palindrome.")
		}
	}
}

File: Python/Palindrome check.py
def main():
    input_string = input("Enter a word: ")
    
    char_array = list(input_string)
    char_array.reverse()
    
    reversed_string = ''.join(char_array)
    
    is_palindrome = input_string == reversed_string
    
    if is_palindrome:
        print("The input is a palindrome.")
    else:
        print("The input is not a palindrome.")

if __name__ == "__main__":
    main()
